types/tress/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
 	"extends": "dtslint/dt.json",
 	"rules": {
-		"space-before-function-paren": false
+		"space-before-function-paren": false,
+		"strict-export-declare-modifiers": false
```

Comment:
What are you trying to suppress here?

Reply:
@DanielRosenwasser, without the `strict-export-declare-modifiers` suppression linting fails on code like below.
https://github.com/sindilevich/DefinitelyTyped/blob/7cc27e8a787a9c77fc242ff6131f8fd88eb0311c/types/tress/index.d.ts#L8